src/trackExtensions.ts

```diff
@@ -1,68 +1,184 @@
 import { Extension, getExtensionHash, getLogs } from '@colony/colony-js';
-import { extensionSpecificEventsListener } from './eventListener';
 
 import networkClient from './networkClient';
-import { verbose } from './utils';
-
-/**
- * Defines extensions for which we want to track ExtensionInitialised events
- * Note: If an extension config in CDapp doesn't specify initialization params,
- * it is considered enabled straight after installation and we don't need
- * to track ExtensionInitialised events for it (e.g. OneTxPayment)
- */
-const supportedExtensionIds = [getExtensionHash(Extension.VotingReputation)];
+import {
+  deleteExtensionFromEvent,
+  isExtensionDeprecated,
+  isExtensionInitialised,
+  mapLogToContractEvent,
+  toNumber,
+  verbose,
+  writeExtensionFromEvent,
+  writeExtensionVersionFromEvent,
+} from './utils';
+import { SUPPORTED_EXTENSION_IDS } from './constants';
+import stats from '../run/stats.json';
+import { extensionSpecificEventsListener } from './eventListener';
 
 export default async (): Promise<void> => {
-  verbose('Fetching already installed extensions');
+  const latestBlock = stats.latestBlock ?? 0;
```

Comment:
Theoretically you can never be at 0, as a chain starts at block 1, but this is more or less irrelevant here, as it won't really change anything